### DISCUSSION TITLE
default open to edit in local

### DISCUSSION THREAD

**Discussion by ajaffrey** [Natizyskunk/vscode-sftp#261] [General]
Is it possible to default opening a file to edit in local rather than right click?

**Comment by NyaPPuu** [comment]
Yes.
I think this is the option you're looking for.
https://github.com/Natizyskunk/vscode-sftp/wiki/Setting#downloadwhenopeninremoteexplorer

**Reply by ajaffrey** [reply]
that does it.. thanks!

**Reply by santosfranklin** [reply]
Excelente me ajudou muito!
